core/src/test/java/org/testcontainers/images/ImagePullPolicyTest.java

```diff
@@ -74,7 +74,7 @@ public void setUp() {
     @Test
     public void pullsByDefault() {
         try (
-            GenericContainer<?> container = new GenericContainer<>(imageName)
+            GenericContainer<?> container = new GenericContainer(new DockerImageName(imageName))
```

Comment:
nit: `imageName` field can be changed to `DockerImageName`, to make tests' bodies a little bit smaller

Reply:
Done